### DISCUSSION TITLE
How to implement multiple clicks on exe to open multiple applications

### DISCUSSION THREAD

**Discussion by victory-zz** [wailsapp/wails#3339] [Q&A]
### Description

I need to implement an application built through wakes build that can open multiple applications by clicking exe

### To Reproduce

null

### Expected behaviour

Clicking on exe can open multiple applications, similar to opening an idea, which can have many windows

### System Details

```shell
window10
```

**Comment by leaanthony** [comment]
Not sure what you mean.

**Comment by victory-zz** [comment]
> Not sure what you mean.

After my wails program is built, it is an application similar to the golang development tools of the JetBrains family.so,i want to implement,open project in new window.You can see that the application has multiple windows in the Windows desktop taskbar
![image](https://github.com/wailsapp/wails/assets/64843745/00c16048-82a0-4b15-905a-6b584db892c3)

**Comment by leaanthony** [comment]
This is a question, not a bug. Moving to discussions.
